tests/drivers/dac/dac_loopback/testcase.yaml

```diff
@@ -1,6 +1,5 @@
 common:
-  tags: dac drivers userspace
+  tags: dac drivers
 tests:
   drivers.dac:
-    platform_whitelist: frdm_k64f nucleo_l073rz nucleo_l152re twr_ke18f
```

Comment:
This won't be sufficient.
In `tests/drivers/dac/dac_loopback/src/test_dac.c` there is some board specific configuration code, that ends up in:
```
#else
#error "Unsupported board."
#endif
```
So I think that with current implementation, the platform_withelist should stand.

Reply:
this depends on dac already, so that should be fine.

Reply:
So each board declaring dac support will have to go and edit this test_dac.c, but maybe at the end someone will try to find a smarter test implementation... ok